docs/reference/indices/aliases.asciidoc

```diff
@@ -271,6 +271,27 @@ POST /_aliases
 <2> The index we should have added
 <3> `remove_index` is just like <<indices-delete-index>>
 
+In case the old index shouldn't be deleted to keep serving the 
+in-flight requests, it is also possible to use `remove` action instead:
+
+[source,console]
+--------------------------------------------------
+PUT test     <1>
+PUT test_2   <2>
+POST /_aliases
+{
+    "actions" : [
+        { "remove": { "index": "test", "alias": "test" } },  <3>
+        { "add":  { "index": "test_2", "alias": "test" } }   <4>
+    ]
+}
+--------------------------------------------------
+
+<1> An index which is currently active
+<2> The index we want to swap with
+<3> `remove` will dissociate `test` index from `test` alias
+<4> `add` will associate `test_2` index with `test` alias
+
```

Comment:
I'm not exactly sure what you're attempting in this example. As written, it does not work because you're attempting to add an alias `test` with the same name as an existing index which is not permitted.